spec/models/user_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  it "has attributes" do
+    User.find_or_create_by_auth({'info' => {uid: 1, 
+                                            name: "fake name", 
+                                            image: "fakeurl.com/image.gif", 
+                                            email: 'example@gmail.com'}})
```

Comment:
Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.